task/requirements.txt

```diff
@@ -1,4 +1 @@
--e git+https://github.com/digital-land/pipeline.git#egg=digital-land
-click==8.0.3
-psycopg2-binary==2.9.3
-requests
\ No newline at end of file
+-r requirements/requirements.txt
```

Comment:
Should we just delete this file? seems unnecessary will need to change the docker build and the read me

Reply:
I've reverted to the old way, so this file will be used